CONTRIBUTING.md

```diff
@@ -34,7 +34,7 @@ To send us a pull request, please:
 2. Modify the source; please focus on the specific change you are contributing. If you also reformat all the code, it will be hard for us to focus on your change.
 3. Ensure local tests pass.
 4. Commit to your fork using clear commit messages.
-5. Send us a pull request, answering any default questions in the pull request interface.
+5. Send us a pull request, answering any default questions in the pull request interface. To create a changelog entry Markdown file in the `.changelog` directory, you can do so manually or use the [changelogger](https://github.com/smithy-lang/smithy-rs/tree/ysaito/switch-to-new-changelog-format/tools/ci-build/changelogger) CLI.
```

Comment:
The link here doesn't seem to be working for me, might need to change?

Reply:
Thanks for catching! Mistakenly pointed to my ex-branch. 3e250cf points to README of `changelogger` in main (when this PR is merged, that README will display the updated contents).